source/common/vue/form/elements/Checkbox.vue

```diff
@@ -1,16 +1,26 @@
 <template>
-  <div class="form-control cb-group">
-    <label class="checkbox">
-      <input
-        v-bind:id="fieldID"
-        type="checkbox" v-bind:name="name" value="yes"
-        v-bind:checked="value"
-        v-bind:disabled="disabled"
-        v-on:input="$emit('input', $event.target.checked)"
+  <div>
+    <div class="form-control cb-group">
+      <label class="checkbox"
+             v-bind:disabled="disabled"
       >
-      <span class="checkmark"></span>
-    </label>
-    <label v-if="label" v-bind:for="fieldID" v-html="label"></label>
+        <input
+          v-bind:id="fieldID"
+          type="checkbox" v-bind:name="name" value="yes"
```

Comment:
Would you mind putting all of these properties on their own respective line? I've just allowed this in the linter because sometimes something only has those three attributes; but this field has many more, so it's okay to have the two additional lines!